env0/resource_sshkey_test.go

```diff
@@ -0,0 +1,41 @@
+package env0
+
+import (
+	"github.com/env0/terraform-provider-env0/client"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"testing"
+)
+
+func TestUnitSshKeyResource(t *testing.T) {
+	resourceType := "env0_ssh_key"
+	resourceName := "test"
+	resourceFullName := resourceAccessor(resourceType, resourceName)
+	sshKey := client.SshKey{
+		Id:    "id0",
+		Name:  "name0",
+		Value: "Key🔑",
+	}
+
+	testCase := resource.TestCase{
+		ProviderFactories: testUnitProviders,
```

Comment:
FYI `ProviderFactories` factories isn't needed here anymore because `runUnitTests` overrides it anyhow. Relevant for other tests in this PR as well. 
(I know you copied it from my test probably, I removed it in one of my recent merged PRs)

Reply:
fa38ab5